{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/views.py

```diff
@@ -69,7 +68,6 @@ class UserViewSet(
     mixins.ListModelMixin,
     mixins.UpdateModelMixin,
 ):
-    swagger_schema = None
```

Comment:
Removing these for the same reason I mentioned elsewhere.
Yes, for a pure API app we might want to hide these so users reading the docs only get API endpoints they can actually call.
But...by default these docs are for frontend devs consuming the backend, so they shouldn't be hidden.
This has the side effect that for some cases you have to do extra work to get Swagger looking the way you want.